config/locales/article/article-en.yml

```diff
@@ -13,6 +13,7 @@ en:
       selected_subjects_combined: Selected subjects combined with %{title}
     delete:
       must_remove_referring_links: You must remove all referring links before you delete this subject.
+      only_subject_owner_can_delete: Only the Subject Owner Can Delete
```

Comment:
Let's change this to "Only the subject creator or project owner can delete this."